src/ashley/locale/fr_FR/LC_MESSAGES/django.po

```diff
@@ -2,18 +2,20 @@ msgid ""
 msgstr ""
```

Comment:
Note that the django.mo file could be gitignored and calculated in the Dockerfile build as it is duplicate information that is in the .po file?

Reply:
Noted for a future improvement, thanks !